src/helpers/getExternalQuestStatus.ts

```diff
@@ -0,0 +1,34 @@
+import { QuestLogInfo } from '@hyperplay/ui'
+import { ExternalEligibility, Quest } from '@hyperplay/utils'
+import { canClaimLeaderboardReward } from './canClaimReward'
+import dayjs from 'dayjs'
+
+export function getExternalQuestStatus(
+  quest: Quest,
+  externalEligibility: ExternalEligibility | null
+): QuestLogInfo['state'] | undefined {
+  const questEnded = quest.end_date
+    ? dayjs(quest.end_date).isBefore(dayjs())
+    : false
+
+  if (quest.status === 'COMPLETED') {
+    return undefined
+  }
+
+  if (quest.status === 'ACTIVE') {
+    return 'ACTIVE'
+  }
+
+  if (quest.status === 'CLAIMABLE') {
+    if (!questEnded) {
+      return 'ACTIVE'
+    }
```

Comment:
how can a quest be claimable but not ended? doesn't claimable come after completed and when completed is set is the "end_date"?

Reply:
This was a mistake, I'd forgotten to push the commit with the fix.